cfgov/v1/models/base.py

```diff
@@ -149,7 +149,7 @@ def generate_view_more_url(self, request):
         available_tags = [tag[0] for name, tags in form.fields['topics'].choices for tag in tags]
         tags = []
         index = util.get_form_id(activity_log)
-        for tag in self.tags.names():
+        for tag in self.tags.slugs():
             if tag in available_tags:
                 tags.append('filter%s_topics=' % index + urllib.quote_plus(tag))
```

Comment:
this funtionality of appending the proper url parameters to the tag, imo, belongs somewhere else, i.e. in the urls.py.  if this eventually calls a view, it should be placed in the urlpatterns.